File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoArgValueContextRow.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnArgBinderContextRow;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoArgValueContextRow extends StudioDtoArgValueContext
{
  public EnumDefnArgBinderContextRow attribute;

  @Nullable
  public String fromAlias;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoArgValueContextRow.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnArgBinderContextRow
import com.neome.api.meta.base.dto.StudioDtoArgValueContext

interface StudioDtoArgValueContextRow : StudioDtoArgValueContext
{
  val attribute: EnumDefnArgBinderContextRow
  val fromAlias: String?
}